coordinator/tsconfig-build.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": [
```

Comment:
maybe better to use include vs exclude in build config? if we ever add any ts files outside of `/src` and `/tests` will be automatically built (not rly the biggest deal)

Reply:
Depends on whether `include` overrides or adds to the `include` in the file it's extending. I'll look into it

Reply:
It does override. Changed.